### DISCUSSION TITLE
Nested route of categories

### DISCUSSION THREAD

**Discussion by jfsenechal** [nuxt/nuxt#15895] [Questions]
Hello

I'm trying to configure a route with one or more categories

```vuejs
pages
  [[sitename]]
      [category]+
         [slugCategory].vue
```

```vuejs
 <NuxtLink
              :to="{ name: 'site-category-slugCategory', params: {
                category:['parent1','parent2'],
              slugCategory: ['cattodisplay'] }}">
My link
          </NuxtLink>
```

what is the correct syntax

**Comment by ndragun92** [comment]
Do you know how many categories u plan to have and is it always category/category/category and end slug is page slug?

if so u can use
`[...page].vue`
and then handle custom logic inside since this supports unlimited number of routes

**Reply by jfsenechal** [reply]
Thanks but it's not really that I want
It gives a rule with zero or more categories
I would like one or more categories
